docs/docs/References/Schemas_and_Autogenerated_Classes.md

```diff
@@ -201,7 +201,7 @@ Defines properties used to [snapshots](Glossary.md#snapshot).
 
 ### SnapshotDefinition Class
 
-Autogenerated based on the [VirtualSourceDefinition Schema](#virtualsourcedefinition-schema).
+Autogenerated based on the [Snapshot Schema](#snapshotdefinition-schema).
 
 ```python
 class VirtualSourceDefinition:
```

Comment:
not your change but can you change this to snapshot too then? I think it's suppose to be snapshot as well?

Reply:
Oh, you mean the code sample.  Ha ha. I don't know why I didn't notice that too.